src/api/v1alpha1/helpers.go

```diff
@@ -54,7 +54,7 @@ func (dk *DynaKube) ActiveGateImage() string {
 	}
 
 	registry := buildImageRegistry(dk.Spec.APIURL)
-	return fmt.Sprintf("%s/linux/activegate:latest", registry)
+	return fmt.Sprintf("%s/linux/activegate:%s", registry, api.RawTag)
```

Comment:
this while file is not used and should not be updated at all.

we should probably remove these in a follow up